### PR TITLE
Closes #72 - Create a demo1release branch for gtd-timer-server

### LINKED ISSUE #72: Create a demo1release branch for gtd-timer-server

### PR DESCRIPTION
**Pull request by oleksandrlytvyn1223**

### CODE REVIEW

File: ServiceTier/Services/LogInService.cs [hunk 1]
```diff
@@ -61,7 +62,8 @@ private static string GenerateToken(User user)
 
             var tokenString = new JwtSecurityTokenHandler().WriteToken(token);
 
-            return tokenString;
+            return JsonConvert.SerializeObject(new
+            { access_token = tokenString });
```
Comment (roman-fay) [inline ServiceTier/Services/LogInService.cs]: make it one line

Comment (oleksandrlytvyn1223) [reply ServiceTier/Services/LogInService.cs]: @roman-fay done

Comment (roman-fay) [reply ServiceTier/Services/LogInService.cs]: so this is commit into develop, but the change was already committed into release branch?

**Comment by roman-fay** [comment]
To clarify - it's *release* branch that is under code freeze. Any commit to *release* requires review.

File: ServiceTier/Services/LogInService.cs (hunk 1 above)
Comment (oleksandrlytvyn1223) [reply ServiceTier/Services/LogInService.cs]: yes without it sign in doesn't work properly

**Pull request merged**